golang/pkg/builder/container/container_builder_test.go

```diff
@@ -89,19 +90,17 @@ func TestEnvPortsLabelsRestartPolicySettings(t *testing.T) {
 			"LABEL2": "1234",
 		}).
 		WithRestartPolicy(containerbuilder.AlwaysRestartPolicy).
-		WithImage("nginx:latest")
-
-	defer builderCleanup(builder)
+		WithImage("nginx:latest").
```

Comment:
Please use full tag name, starting with docker.io.

Reply:
It was not added by me, but I will change it everywhere then.